tensorflow/lite/kernels/slice_test.cc

```diff
@@ -238,6 +238,25 @@ TEST_P(SliceOpTest, SliceString) {
                                 "2,0,0,0", "2,0,1,0", "2,0,2,0"}));
 }
 
+TEST_P(SliceOpTest, SliceInt64) {
```

Comment:
You'll need to blacklist these 2 new tests @ https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/delegates/nnapi/acceleration_test_list.cc#L330